server/paststream_poller.js

```diff
@@ -0,0 +1,47 @@
+function createPollRoute(channelID) {
+    return `https://holodex.net/api/v2/channels/${channelID}/videos?lang=en&type=stream%2Cplaceholder&include=live_info&limit=24&offset=0&paginated=true`
+}
+
+async function fetchPaststreamPage(channelID) {
+    try {
+        const res = await fetch(createPollRoute(channelID))
+        if (res.status !== 200) {
+            return { error: `HTTP status: ${res.status}`, result: null }
+        }
+        const youtubeJSON = await res.json()
+        return { error: null, result: youtubeJSON }
+    } catch (e) {
+        return { error: e.toString(), result: { items: [] } }
```

Comment:
Please make the result for the error case `null`, since it'll never be used by pollPaststreamStatus.